packages/perseus/src/widgets/interactive-graphs/graphs/circle.tsx

```diff
@@ -30,7 +31,7 @@ export function renderCircleGraph(
 ): InteractiveGraphElementSuite {
     return {
         graph: <CircleGraph graphState={state} dispatch={dispatch} />,
-        interactiveElementsDescription: null,
+        interactiveElementsDescription: CircleGraphDescription({state}),
```

Comment:
I'm slightly surprised that this works. Since `CircleGraphDescription` is calling a hook, I'd expect it to have to be used as a JSX element:

```tsx
<CircleGraphDescription state={state} />
```

If the component calling `renderCircleGraph` does so conditionally, e.g.

```ts
if (graphType === "circle") {
  return renderCircleGraph(state, dispatch)
}
```

And `graphType` changes between renders, this code will break the Rules of Hooks. The `usePerseusI18n` hook will be called on one render and not called on the next.

I think it's working (mostly?) because the graph type rarely changes between renders. It probably only does so in the content editor.

Reply:
Ack! My mistake. I 100% meant to use your code for Point as the blueprint for this, which means I was supposed to write `<CircleGraphDescription state={state} />`.